d2bs/kolbot/libs/bots/MFHelper.js

```diff
@@ -211,15 +210,17 @@ MainLoop:
 
 						if (!me.inTown && !Pather.usePortal(null, player.name)) {
 							Town.goToTown();
+							if (Config.MFHelper.HealBetweenCommands) {
```

Comment:
Town.heal checks config options via "needHealing" therefore, separate config not needed for healbetweencommands

Reply:
What if, as @DarkHorseDre stated, you don't want the helper to heal because you decide it won't have time to, but keep the behavior of `Town.heal` in other scripts ?
The thing is, this is an optional config which default value is `false`, but when `true`, it will heal if needed, based on `needHealing`. In other scripts we don't need this kind of config because Town chores are executed between each scripts.
While writting these lines, my thought is now : Town chores are executing between each scripts, which is not the case for MFHelper which may do all the game without doing chores. What about pushing this behavior further and do
```
if (Config.MFHelper.DoChoresBetweenCommands) {
	Town.doChores();
}
```
?